v2/pkg/core/inputs/hybrid/hmap.go

```diff
@@ -93,6 +99,10 @@ func (i *Input) initializeInputSources(options *types.Options) error {
 func (i *Input) scanInputFromReader(reader io.Reader) {
 	scanner := bufio.NewScanner(reader)
 	for scanner.Scan() {
+		if isCIDR(scanner.Text()) {
```

Comment:
We can use [`iputil.IsCIDR`](https://github.com/projectdiscovery/iputil/blob/5940e7c42cb040deed0dcda9ed41560f70ce2e16/iputil.go#L74)

Reply:
Updated the function to use `iputil.IsCIDR` as per your suggestion.